### DISCUSSION TITLE
small issue inserting new waypoint far away from any road

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#456] [Q&A] [answered]
After setting a waypoint in the midth of nowhere the number mentioned in the error message is wrong. After just initiate a re-calculation the message is correct.
<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/170406ce-7566-4aa2-9400-2f1fecdfb087"/>  <img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/32be897a-b1ec-4066-8d85-2f760924155a"/>  <img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/919bf70a-4be8-4ddb-bea2-a7c8f2e8a0fa"/>

No big issue ... just something I noticed.

**Comment by devemux86** [comment]
@WolfgangZ42 thanks for the report.

This is with the intelligent route calculation?

**Reply by WolfgangZ42** [reply]
yes, it is.

**Comment by devemux86** [comment, marked as answer]
Fixed in [Cruiser 3.0.41](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
